nuspec/nuget/Cake.CakeMail.nuspec

```diff
@@ -3,15 +3,18 @@
   <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <id>Cake.CakeMail</id>
     <version>0.0.0</version>
-    <authors>gep13</authors>
-    <owners>gep13</owners>
+    <authors>jericho</authors>
+    <owners>jericho</owners>
     <description>Cake CakeMail AddIn.</description>
-    <summary>Cake AddIn that extends Cake with ability to execute the gem command line utility.</summary>
+    <summary>Cake AddIn that extends Cake with ability to send emails via CakeMail.</summary>
     <licenseUrl>https://github.com/cake-contrib/Cake.CakeMail/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/cake-contrib/Cake.CakeMail/</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <copyright>Copyright (c) Cake Contributions 2016 - Present</copyright>
     <tags>Cake, Script, Build, CakeMail</tags>
+    <dependencies>
+      <dependency id="CakeMail.RestClient" version="5.0.131" />
```

Comment:
Same comment as SendGrid.  RestSharp is not an MIT license, so we will likely need to include their license agreement in the package as well.